.sbtopts

```diff
@@ -0,0 +1,6 @@
+-J-XX:MaxMetaspaceSize=4G
```

Comment:
though I'm generally not a fan of .sbtopts because it impacts dev environments (I personally have `SBT_OPTS` set up in my bashrc) , I appreciate you taking the time to set it up considering the complexity of the build and the fact that these might help people who checkout the repo :+1: